src/unix/linux-core.c

```diff
@@ -1015,14 +1025,263 @@ uint64_t uv_get_total_memory(void) {
 }
 
 
-static uint64_t uv__read_cgroups_uint64(const char* cgroup, const char* param) {
-  char filename[256];
+/*
+ * Holds information about a given cgroups subsystem.
+ */
+typedef struct {
+  /* cgroups version, one of CGROUPS_VERSION_*. */
+  uint8_t cgroups_version;
+  /* Path in which param files are found for a subsystem. */
+  char* path;
```

Comment:
style (here ald elsewhere): `char *path`

Reply:
Hasn't the libuv style always been "the star leans left?"

Reply:
Yeah sorry, brainfart.